priv/migrations/20210914100800_index_inner_txs.ex

```diff
@@ -28,8 +27,6 @@ defmodule AeMdw.Migrations.IndexInnerTxs do
     if not from_startup? and :ok != Application.ensure_started(:ae_mdw) do
       IO.puts("Ensure sync tables...")
       SyncSup.init_tables()
-      MdwApp.init(:contract_cache)
-      MdwApp.init(:db_state)
```

Comment:
Needed by the migration because it replays the sync calling `AeMdw.Db.Sync.Transaction.sync_transaction/4`.